alerting/provider/incidentio/incident_io.go

```diff
@@ -0,0 +1,205 @@
+package incidentio
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"net/http"
+	"strconv"
+
+	"github.com/TwiN/gatus/v5/alerting/alert"
+	"github.com/TwiN/gatus/v5/client"
+	"github.com/TwiN/gatus/v5/config/endpoint"
+	"gopkg.in/yaml.v3"
+)
+
+const (
+	restAPIUrl     = "https://api.incident.io/v2/alert_events/http"
+	firingStatus   = "firing"
+	resolvedStatus = "resolved"
+)
+
+var (
+	ErrAlertSourceConfigNotSet = errors.New("alert source config not set.")
```

Comment:
```suggestion
	ErrAlertSourceConfigNotSet = errors.New("alert-source-config-id not set.")
```
As someone who reads lots of error logs, having the exact name of the field which is not set might make for a _slightly_ better error.